docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md

```diff
@@ -12,41 +12,65 @@ isTocNested: false
+1. Install the prerequisites and verify the system resource limits as described in [system configuration](/latest/deploy/manual-deployment/system-config).
+1. You need `ssh` access to the machine and root access (or the ability to run `sudo`; the sudo user can require a password but having passwordless access is desirable for simplicity and ease of use).
+1. Verify that you can `ssh` into this node (from your local machine if the node has a public address).
 
+    ```sh
+    $ ssh -i your_private_key.pem ssh_user@node_ip
+    ```
 
-```
-$ ssh -i your_private_key.pem ssh_user@node_ip
-```
+The following actions will be performed with sudo access.
 
-5. These are the actions that will be performed with sudo access.
-    *   Create the `yugabyte:yugabyte` user + group
-    *   Set home dir /home/yugabyte
-    *   Create the `prometheus:prometheus` user + group
+* Create the `yugabyte:yugabyte` user + group.
+* Set home dir /home/yugabyte.
+* Create the `prometheus:prometheus` user + group.
 
 {{< tip title="Tip" >}}
-In case you’re using the LDAP directory for managing system users, then you can pre-provision yugabyte and prometheus users 
+If you're using the LDAP directory for managing system users, you can pre-provision Yugabyte and Prometheus users: 
 
-*   `yugabyte` user should belong to the `yugabyte` group
-*   Set home dir for yugabyte user (default /home/yugabyte) and ensure the directory is owned by `yugabyte:yugabyte`. The home directory will be used in the cloud provider config step
-*   Prometheus username and the group can be user-defined. You will enter the custom user during the cloud provider configuration setup.
-{{< /tip >}}
+* The `yugabyte` user should belong to the `yugabyte` group.
 
-    * Ability to schedule Cron jobs with Crontab. Cron jobs are used for health monitoring, log file rotation, and system core files cleanup.
+* Set the home dir for the `yugabyte` user (default /home/yugabyte) and ensure the directory is owned by `yugabyte:yugabyte`. The home directory will be used during cloud provider configuration.
+    
+* The Prometheus username and the group can be user-defined. You will enter the custom user during cloud provider configuration.
+{{< /tip >}}
 
+* Ensure you can schedule Cron jobs with Crontab. Cron jobs are used for health monitoring, log file rotation, and system core files cleanup.
 
 {{< tip title="Tip" >}}
-For any 3rd party cron scheduling tools, you can add these cron entries and disable Crontab. Disabling Crontab will create alerts after the universe creation that can be ignored. But you need to ensure cron jobs are set appropriately  for the platform to work as expected. 
+For any 3rd party cron scheduling tools, you can add these cron entries and disable Crontab. Disabling Crontab will create alerts after the universe creation that can be ignored. But you need to ensure cron jobs are set appropriately for the platform to work as expected. 
```

Comment:
This comment is applicable to most of the content: present tense should be used (future tense should be avoided unless absolutely necessary).
And this sentence is off/confusing: "Disabling Crontab will create alerts after the universe creation that can be ignored."